src/dbus.c

```diff
@@ -949,15 +955,29 @@ gboolean dbus_cb_dunst_Properties_Set(GDBusConnection *connection,
                                       GError **error,
                                       gpointer user_data)
 {
+        int targetPauseLevel = -1;
         if (STR_EQ(property_name, "paused")) {
-                dunst_status(S_RUNNING, !g_variant_get_boolean(value));
+                if (g_variant_get_boolean(value)) {
+                        targetPauseLevel = 100;                 
```

Comment:
It may be useful to define a variable for `MAX_PAUSE_LEVEL`, set it to 10 and use it here.